app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -69,8 +70,13 @@ class PasswordFragment : Fragment() {
         swipeRefresher = binding.swipeRefresher
         swipeRefresher.setOnRefreshListener {
             if (!PasswordRepository.isGitRepo()) {
-                Snackbar.make(binding.root, getString(R.string.clone_git_repo), Snackbar.LENGTH_SHORT)
-                    .show()
+                Snackbar.make(binding.root, getString(R.string.clone_git_repo), Snackbar.LENGTH_INDEFINITE)
+                        .setAction(R.string.clone_button) {
+                            val intent = Intent(context, GitServerConfigActivity::class.java)
+                            intent.putExtra(BaseGitActivity.REQUEST_ARG_OP, BaseGitActivity.REQUEST_CLONE)
+                            startActivityForResult(intent, BaseGitActivity.REQUEST_CLONE)
```

Comment:
Is the snackbar dismissed automatically when a new activity is started?

Reply:
Once the action is triggered Snackbar is dismissed.